package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-find-rules",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
I can do that using separate PRs in the future, if you are ok with the additional approvals that become necessary.
At least we could "squash & merge" most PRs again, keeping the commit history a bit cleaner.